device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,285 @@
+{
+    "chassis": {
+        "name": "MSN2700",
+        "components": [],
+        "fans": [
+            {
+                "name": "fan1"
+            },
+            {
+                "name": "fan2"
+            },
+            {
+                "name": "fan3"
+            },
+            {
+                "name": "fan4"
+            },
+            {
+                "name": "fan5"
+            },
+            {
+                "name": "fan6"
+            },
+            {
+                "name": "fan7"
+            },
+            {
+                "name": "fan8"
+            }
+        ],
+        "fan_drawers": [
+            {
+                "name": "drawer1",
+                "num_fans": 2
+            },
+            {
+                "name": "drawer2",
+                "num_fans": 2
+            },
+            {
+                "name": "drawer3",
+                "num_fans": 2
```

Comment:
Rather than `num_fans`, I believe we should change this to:

```
"fans": [
    {
        "name": "fan_5"
    },
    {
        "name": "fan_6"
    }
]
```

@keboliu: Please confirm.

Reply:
same as above https://github.com/Azure/sonic-buildimage/pull/5189/files#r491907520

Reply:
fixed